server/test/controllers/ti/TrustedIntermediaryControllerTest.java

```diff
@@ -75,8 +75,7 @@ public void testUpdateClientInfo_ThrowsApplicantNotFoundException()
     account.setManagedByGroup(tiGroup2);
     account.save();
 
-    assertThat(testApplicant.get().getApplicantData().getDateOfBirth().get().toString())
-        .isEqualTo("2022-07-18");
+    assertThat(testApplicant.get().getDateOfBirth().get().toString()).isEqualTo("2022-07-18");
```

Comment:
suggestion (non-blocking): keep both tests in here for now and remove the WKP test cases when you remove those code paths.